src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -360,6 +360,9 @@ internal void FunctionEnvironmentReloadResponse(FunctionEnvironmentReloadRespons
             _workerChannelLogger.LogDebug("Received FunctionEnvironmentReloadResponse from WorkerProcess with Pid: '{0}'", _rpcWorkerProcess.Id);
 
             LogWorkerMetadata(res.WorkerMetadata);
+
+            _workerConfig.Description.DefaultRuntimeVersion = _workerConfig.Description.DefaultRuntimeVersion ?? res.WorkerMetadata.RuntimeName;
```

Comment:
I [see ](https://github.com/Azure/azure-functions-host/blob/0266f360d578f06901b5021152ac7fb18d0ec160/test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerConfigTests.cs#L514)the `DefaultRuntimeVersion` prop stores the version value (ex:3.7) for the language (ex: python). But this assignment is setting it to the runtime name (python). Is that expected?

Reply:
Have added a new runtime name property.